@shared/api/internal.ts

```diff
@@ -581,6 +587,46 @@ export const saveSettings = async ({
   return response;
 };
 
+export const changeNetwork = async (
+  networkName: string,
+): Promise<NetworkDetails> => {
+  let networkDetails = MAINNET_NETWORK_DETAILS;
+
+  try {
+    ({ networkDetails } = await sendMessageToBackground({
+      networkName,
+      type: SERVICE_TYPES.CHANGE_NETWORK,
+    }));
+  } catch (e) {
+    console.error(e);
+  }
+
+  return networkDetails;
+};
+
+export const addCustomNetwork = async (
+  customNetwork: CustomNetwork,
+): Promise<{
+  networkDetails: NetworkDetails;
+  networksList: NetworkDetails[];
```

Comment:
Add a network to the list of networks available. Returns the new list of networks and the current network